src/app.module.ts

```diff
@@ -4,6 +4,8 @@ import { DatabaseModule } from './database/database.module';
 import { UserModule } from './user/user.module';
 import { APP_INTERCEPTOR } from '@nestjs/core';
 import { AuthModule } from './auth/auth.module';
+import { InstructorModule } from './instructor/instructor.module';
+import { CourseModule } from './course/course.module';
```

Comment:
at this level, meaning all import from other module to other, we need to use absolute import